src/modules/site-v2/base/views/strains.py

```diff
@@ -324,7 +327,8 @@ def order_page_index():
     'title': "Order Summary",
     'cartItems': cartItems,
     'totalPrice': totalPrice,
-    'form': form
+    'form': form,
+    'EULA_url': get_blob(MODULE_SITE_BUCKET_PRIVATE_NAME, 'EULA_for_CaeNDR_website_Dr_Andersen_1-25-24.md').public_url

```

Comment:
consider moving this value ('EULA_for_CaeNDR_website_Dr_Andersen_1-25-24.md') to the `.env` so that we can swap without having to re-deploy code. With the EULAs, I think everytime there is a change to the text of the EULA, we need to preserve the older ones and timestamp when new ones were deployed. This is the case where someone agrees to a EULA from 6 months ago, and today's EULA has changed. The 'contract' is that the user agreed not to today's EULA, but rather to the one from 6 months ago -- therefor, we should preserve historical record of all EULAs moving forward.

Reply:
and as a side effect, when we `track` that the user agreed to the EULA, we should reference *which* EULA version they agreed to.